firebase/package.json

```diff
@@ -35,6 +35,7 @@
   },
```

Comment:
This is a good example that I've exemplified in our discussion in https://github.com/olmps/memo/pull/255